packages/nocms-plugin-directory-resources/src/rollupDefaults.js

```diff
@@ -0,0 +1,52 @@
+import babel from 'rollup-plugin-babel';
+import commonjs from 'rollup-plugin-commonjs';
+import nodeResolve from 'rollup-plugin-node-resolve';
+import replace from 'rollup-plugin-replace';
+import { uglify } from 'rollup-plugin-uglify';
+
+const inputOptions = {
+	plugins: [
+		babel({
+			presets: [
```

Comment:
What. Are `presets` and `plugins` two-dimensional arrays? It looks weird :-D

Reply:
Yes! I got confused as well and had the wrong encapsulation, but it checks out. The string first, options later convention is a bit wacky, should be a key, but is common for babel/other build tools

Reply:
Babel sucks that way